components/homme/src/theta-l/CMakeLists.txt

```diff
@@ -50,22 +52,13 @@ SET(THETAL_SRCS_F90
   ${UTILS_SHARE_DIR}/shr_spfn_mod.F90
 )
 
-IF (HOMME_USE_ARKODE)
-  SET(THETAL_SRCS_C
-    ${ARKODE_DIR}/nvector_external.h
-    ${ARKODE_DIR}/nvector_external.c
-    ${ARKODE_DIR}/column_linsol.h
-    ${ARKODE_DIR}/column_linsol.c
-  )
-ENDIF ()
-
 # If the user specified a file for custom compiler options use those
 IF (DEFINED THETA_CUSTOM_FLAGS_FILE)
   setCustomCompilerFlags(THETA_CUSTOM_FLAGS_FILE THETAL_SRCS_F90)
 ENDIF ()
 
 # Make SRCS global so the tests can access it
-SET(EXEC_SOURCES ${THETAL_SRCS} ${THETAL_SRCS_C} ${THETAL_SRCS_F90} )
```

Comment:
was this deletion intentional?

Reply:
Yes, the `THETA_L_SRCS_C` list only contained files that wrapped the ARKODE C functions in a way that they could be called from Fortran.  Essentially, ARKODE now provides Fortran wrappers so we don't need to maintain those "THETA_L_SRCS_C" files in HOMME.